site/src/components/Playground.js

```diff
@@ -207,6 +213,27 @@ const Editor = ({ onChange, initialCode, code, language }) => {
         <Button ref={copyButtonRef} variant="secondary" size="small" onClick={copy}>
           Copy code
         </Button>
+        {copyState === copySuccessState && (
+          <Notification
+            type="success"
+            label="The example code is copied to clipboard."
+            position="bottom-right"
+            autoClose
+            displayAutoCloseProgress={false}
+            onClose={() => setCopyState('')}
+          />
+        )}
+        {copyState === copyErrorState && (
+          <Notification
+            type="error"
+            label="Code copy failed."
+            position="bottom-right"
+            displayAutoCloseProgress={false}
+            dismissible
```

Comment:
Is there a reason why the notifications differ in case of success vs error? I mean the error has the close button, and it won't automatically close?

Reply:
I think error messages are something we require users to see because the copy failed, and they either have to try it again or try to copy the code in some other way. If they autoclose, users might miss them in some cases.
I would rather remove the autoclose from success message, but then it would require two clicks to continue when everything went smoothly.

Reply:
I think the error message could suggest to manually copy the code once the button functionality fails 🤔

Reply:
Now I think the success and error texts are too heavy since it's in bold 😅 It looks a bit tacky in my opinion. I fiddled with it and I think it looks better with shorter label and using the children prop for more information. Try these ones for size and see what you think:
- on success:`label="Code copied" children={<span>Example code was copied to clipboard.</span>}`
- on error: `label="Code copying failed" children={<span>Try again or copy the code using the clipboard.</span>}`

Reply:
- [x] update notification texts